lib/matplotlib/pyplot.py

```diff
@@ -3787,4 +3791,60 @@ def spectral():
     if im is not None:
         im.set_cmap(cm.spectral)
 
+
+# This function was autogenerated by boilerplate.py.  Do not edit as
+# changes will be lost
+def magma():
```

Comment:
I thought these functions were not supposed to be created any longer?  Only the ones included for Matlab compatibility should exist, and all others should be used as `set_cmap('magma')`?  See https://github.com/matplotlib/matplotlib/pull/889#discussion_r867101